src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -6203,84 +6204,81 @@ namespace ZoneTempPredictorCorrector {
         // Sum all system air flow: SumSysMCp, SumSysMCpT
         // Check to see if this is a controlled zone
         ControlledZoneAirFlag = Zone(ZoneNum).IsControlled;
+        if (CorrectorFlag) {
```

Comment:
So if we are just in predictor, we skip a bunch of stuff.  I'm fine with that conceptually, and if CI is showing no changes in results then that's enough confirmation.